cyences_app_for_splunk/default/data/models/Cyences_Vulnerabilities.json

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
Can we enable the acceleration of the data-model by default?

Reply:
The default acceleration is not allowed by appinpect.